app/assets/stylesheets/welcome.scss

```diff
@@ -136,3 +132,20 @@ a {
 .octo {
   margin-top: 0px;
 }
+
+#search-dropdown {
+  margin-top: 85px;
+}
+
+.list-group-item {
+  background: #38d5fb;
```

Comment:
Color literals like `#38d5fb` should only be used in variable declarations; they should be referred to via variable everywhere else.